main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"encoding/base64"
 	"fmt"
-	"log"
+	log "github.com/Sirupsen/logrus"
```

Comment:
yea my IDE freaks out about it without the alias and trys to import the built-in still...haven't figured out how to turn that off yet.